src/loaders/svg/tvgSvgPath.cpp

```diff
@@ -44,12 +44,14 @@ static bool _parseNumber(char** content, float* number)
 }
 
 
-static bool _parseLong(char** content, int* number)
+static bool _parseFlag(char** content, int* number)
```

Comment:
I agree that the argument can be a float type and we need to support it.
However, you can easily parse the float type using strof.
And I think you can do typecast it to int.
_parseNumberToInt(char**, int* n)
{
n = (int)strof( , );
...
}
What do you think?

Reply:
Well, not exactly. In svg spec (w3.org), the info about large-arc and sweep are:
- these two are flags
- we know what to do if their values are '0' or '1'
- in chapter 9.5.1 we have info, that "Arbitrary numerical values are permitted for all elliptical arc parameters (**other than the boolean flags**), ..."

At first I thought that we could allow values 1.0 and 0.0, but after some consideration I think that the safest approach would be to check for the exact values and type for these flags. If in a path there will be a mistake, like a wrong order of arguments after 'A' cmd, we won't be able to catch this mistake using typecasting.

Reply:
Sorry I completely misunderstood your intentions.